Source/WebCore/rendering/svg/RenderSVGResourceMasker.cpp

```diff
@@ -186,4 +179,35 @@ void RenderSVGResourceMasker::removeReferencingCSSClient(const RenderElement& cl
         m_masker.remove(renderer);
 }
 
+bool RenderSVGResourceMasker::drawContentIntoContext(GraphicsContext& context, const FloatRect& objectBoundingBox)
+{
+    // Eventually adjust the mask image context according to the target objectBoundingBox.
+    AffineTransform maskContentTransformation;
+
+    if (maskElement().maskContentUnits() == SVGUnitTypes::SVG_UNIT_TYPE_OBJECTBOUNDINGBOX) {
+        maskContentTransformation.translate(objectBoundingBox.location());
+        maskContentTransformation.scale(objectBoundingBox.size());
+    }
+
+    // Draw the content into the ImageBuffer.
+    checkedLayer()->paintSVGResourceLayer(context, maskContentTransformation);
+    return true;
+}
+
+bool RenderSVGResourceMasker::drawContentIntoContext(GraphicsContext& context, const FloatRect& destinationRect, const FloatRect& sourceRect, ImagePaintingOptions options)
+{
+    GraphicsContextStateSaver stateSaver(context);
+
```

Comment:
I think you can omit some newlines here for readability.

Reply:
Fair enough, removed some.